internal/upstreamwatcher/upstream_watcher.go

```diff
@@ -17,6 +18,14 @@ func WatchUpstreams(gloo storage.Interface,
 			upstreams <- newList
 		}
 	}
+
+	// get the party started
+	initialList, err := gloo.V1().Upstreams().List()
+	if err != nil {
+		return nil, errors.Wrap(err, "getting initial list of upstreams")
+	}
+	go syncFunc(initialList, nil)
```

Comment:
is this still needed after  25 worker change?

Reply:
hm possibly not, i didnt chage. i can remove if you'd like, or we can keep it. it didn't seem to help anything anyways

Reply:
If not needed let's remove.. less code the better